README.md

```diff
@@ -42,6 +42,13 @@ Additionally, `ode23s` solver supports
 
 There are also fixed step Runge-Kutta and Rosenbrock solvers available.
 
+# Examples
+The examples directory contain a few notebooks that show how to get started. You can also see them here:
+* [Simple differential equation](http://nbviewer.jupyter.org/github/pjpmarques/ODE.jl/blob/master/examples/Simple_Differential_Equation.ipynb)
+* [Lorenz Attractor](http://nbviewer.jupyter.org/github/pjpmarques/ODE.jl/blob/master/examples/Lorenz_Attractor.ipynb)
+* [Terminal Velocity](http://nbviewer.jupyter.org/github/pjpmarques/ODE.jl/blob/master/examples/Terminal_Velocity.ipynb)
+
```

Comment:
It might be (but isn't necessarily) nice to first merge this, though, and then when the notebooks exist in ODE.jl, refer to them with a specific hash rather than `master`. This would ensure that links aren't broken by mistake if/when the notebooks are (re)moved, but on the other hand it would require manual update of all links when the notebooks are changed, even if the location did not change; and furthermore, that manual change would not be possible to make before the changes to the notebooks have been committed.